ajax/libs/jquery.cycle2/package.json

```diff
@@ -19,5 +19,37 @@
     "type": "git",
     "url": "https://github.com/malsup/cycle2"
   },
-  "license": "MIT"
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/malsup/cycle2.git",
+    "basePath": "src",
```

Comment:
There is a `build` folder but you didn't use it, any reason why?

https://github.com/malsup/cycle2/tree/master/build